Monika After Story/game/script-greetings.rpy

```diff
@@ -371,3 +371,52 @@ label monikaroom_greeting_post:
         set_keymaps()
     return
 
+label_greeting_youarereal:
+default persistent.you = True
+    python:
+        try: 
+            renpy.file("/characters/" + persistent.playername + ".chr")
+            persistent.you = True
+        except IOError:
+            persistent.you = False
+    m 1b "[player]! Great to see you!"
+
+if persistent.you:
+    jump youreal
+else:
+    return
+
+    
+label youreal:
```

Comment:
You don't need a separate label for the dialogue here unless you're expecting two separate renpy flows (or some special case, in which case, please explain). Move the dialogue you've added into the greeting label at the appropriate spot.